### ISSUE TITLE
Removing test files causes the app to crash

### ISSUE THREAD

**Issue by Sanjo** [meteor-velocity/node-soft-mirror#9]
`packages/tests-proxy/package.js` must be removed manually each time.

The error is something like
```
=> Errors prevented startup:                  

While Building package `velocity:test-proxy`:
error: File not found: tests/jasmine/server/integration/sample/spec/SpecMatchers.js
error: File not found: tests/jasmine/server/integration/sample/src/Song.js
error: File not found: tests/jasmine/server/integration/sample/spec/PlayerSpec.js
error: File not found: tests/jasmine/server/integration/sample/src/Player.js
```

**Comment by guilhermedecampo** [comment]
Also notice this bug!

Deleting folders can cause this issue.
In my case I just delete the integration folder on client's folder.

**Comment by bobiblazeski** [comment]
This bug is still happening, I just deleted the file tests/mocha/server/sampleServerTest.js
While building package velocity:test-proxy:
error: File not found: tests/mocha/server/sampleServerTest.js

**Comment by typ90** [comment]
Any update on this?

**Comment by Sanjo** [comment]
I don't think so. The workaround is still to delete the folder `packages/tests-proxy/` and restart the app.

**Comment by AdamBrodzinski** [comment]
+1 Thanks @Sanjo  This happened to me after deleting the sample specs while the app was running. Deleting tests-proxy did the trick. :beers:

**Comment by adamdawkins** [comment]
+1 @Sanjo you beauty, the workaround is good enough for me - was going insane trying to solve this!

**Comment by aj07mm** [comment]
```
The workaround is still to deleting the folder packages/tests-proxy/ and restart the app.
```
+1 Worked here
Wasted some time trying to figure out why the new tests were not running. 
Thks!

**Comment by jerefrer** [comment]
Still happening to me, any news on this ?

**Comment by spencern** [comment]
Still happening to me as well. Might be worth leaving this issue open until it's fixed for people who are searching. Workaround still fixes in the meantime.

**Comment by carlosbaraza** [comment]
:+1:

**Comment by pinxue** [comment]
A bit more specific, remove api.add_files() referencing the file in packages/test-proxy/package.js , like

Package.onUse(function (api) {
    api.use("coffeescript", ["client", "server"]);
    api.add_files("tests/mocha/client/hellotest.js",["client"]);
});

I guess it can be fixed.

**Comment by ghost** [comment]
We have released velocity:core 0.6.0 that no longer uses node-soft-mirror. The newest versions of mike:mocha and sanjo:jasmine also no longer use the node-soft-mirror.

After you have updated, you need to delete the package/tests-proxy folder.
